pss/crypto/utils.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"sync"
+
+	ethCrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+// Utils contains utility methods for tests. Generates and stores asymmetric keys
+type Utils interface {
```

Comment:
nothing here suggests that this is assymetric crypto support

Reply:
To be clear: Util is _only_ used for testing stuff, right?

Reply:
@zelig I'm not so sure I'd get too hung up about what the interface looks like right now. The final interfaces need a SWIP proposal and good thinking through. I don't see a problem making an intermediate pass as long as the APIs everyone currently are using for `pss` are not affected.

Reply:
Was not my intention. We detected some calls to whisper functions only used from tests. We didn't want to polute the new crypto API with those calls, that's why we separated. It is just accidental that all the utilities are asymmetric functions. I can foresee other methods here and maybe some of them could be symmetric, hashing related....

Reply:
> To be clear: Util is _only_ used for testing stuff, or?

Right, only for testing. I will be more specific in the comments to avoid confusions.